File: src/main/java/xyz/chunkstories/api/input/InputsManager.java
//
// This file is a part of the Chunk Stories API codebase
// Check out README.md for more information
// Website: http://chunkstories.xyz
//

package xyz.chunkstories.api.input;

import java.util.Iterator;

public interface InputsManager {
	public Input getInputByName(String inputName);

	public Input getInputFromHash(long hash);

	public Iterator<Input> getAllInputs();
}

File: src/main/java/xyz/chunkstories/api/input/InputsManager.kt
//
// This file is a part of the Chunk Stories API codebase
// Check out README.md for more information
// Website: http://chunkstories.xyz
//

package xyz.chunkstories.api.input

interface InputsManager {

    val allInputs: Collection<Input>

    fun getInputByName(inputName: String): Input?
    fun getInputFromHash(hash: Long): Input?
}
